assets/js/components/TourTooltips.test.js

```diff
@@ -215,8 +235,10 @@ describe( 'TourTooltips', () => {
 		beforeEach( () => mockTrackEvent.mockClear() );
 
 		it( 'tracks all events for a completed tour', async () => {
-			const { getByRole } = renderTourTooltipsWithMockUI( registry );
-			await getByRole( 'alertdialog' );
```

Comment:
`await getByRole` should never be allowed as `getBy` is not async

https://www.npmjs.com/package/eslint-plugin-testing-library lints for this kind of stuff *but* it doesn't recognise our WordPress imports out the box